Makefile

```diff
@@ -14,6 +15,9 @@ deps:
 build: clean deps
 	$(COMMONENVVAR) $(BUILDENVVAR) godep go build -o kube-state-metrics 
 
+crossbuild:
```

Comment:
@brancz should this be added to 1.0 milestone to support build multi arches binary releases?